src/details/ArborX_DetailsBatchedQueries.hpp

```diff
@@ -48,16 +48,24 @@ struct BatchedQueries
   // the end.  We decided to keep reversePermutation around for now.
 
   template <typename ExecutionSpace, typename Predicates,
-            typename SpaceFillingCurve>
+            typename BoundingVolume, typename SpaceFillingCurve>
```

Comment:
That does not feel right to enable anything else than boxes here.

Reply:
I did not want to proliferate `expand()` calls before calling this function. But I see your point.